javav2/example_code/lookoutvision/src/main/java/com/example/lookoutvision/DescribeDataset.java

```diff
@@ -0,0 +1,79 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+// snippet-start:[lookoutvision.java2.describe_dataset.complete]
+
+package com.example.lookoutvision;
+
+import software.amazon.awssdk.services.lookoutvision.LookoutVisionClient;
+import software.amazon.awssdk.services.lookoutvision.model.DatasetDescription;
+import software.amazon.awssdk.services.lookoutvision.model.LookoutVisionException;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+// Describes an Amazon Lookout for Vision dataset.
+public class DescribeDataset {
+
+    public static final Logger logger = Logger.getLogger(DescribeDataset.class.getName());
+
+    public static void main(String[] args) throws Exception {
+
+        String projectName = null;
+        String datasetType = null;
+
+        final String USAGE = "\n" +
+                "Usage:\n" +
+                "    DescribeProject <project_name> <dataset_type> \n\n" +
+                "Where:\n" +
+                "    project_name - the name of the project that you want to describe.\n" +
+                "    dataset_type - the type (train or test) of the dataset that you want to describe.\n\n";
```

Comment:
"    dataset_type - The type of the dataset that you want to describe (train or test) .\n\n";

Reply:
Fixed.